src/inverseRobot.js

```diff
@@ -24,7 +24,16 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const copy = {};
+
+  for (const x in robot) {
```

Comment:
```suggestion
  const copiedRobot = {};

  for (const part in robot) {
```

Try to give more meaningful names to variables

Reply:
Okay, thanks!